API/application/iotjs.py

```diff
@@ -103,6 +103,15 @@ def __apply_stack_patches(self, device, revert=False):
             iotjs_stack_nuttx_patch = utils.join(paths.PATCHES_PATH, 'iotjs-stack-nuttx.diff')
             utils.patch(paths.IOTJS_PATH, iotjs_stack_nuttx_patch, revert)
 
+    def __apply_coverage_patches(self, device, revert=False):
+        if device.get_type() == 'artik053':
+            jerry_coverage_patch = utils.join(paths.PATCHES_PATH, 'jerry-debugger.diff')
+            utils.patch(paths.IOTJS_JERRY_PATH, jerry_coverage_patch, revert)
+            utils.execute(paths.IOTJS_JERRY_PATH, 'git', ['add', '-u'])
```

Comment:
Please remove this line because it's enough to have only one `git add -u` command per sub-project.